Sources/XCRemoteCache/Dependencies/DependencyProcessor.swift

```diff
@@ -58,11 +58,11 @@ class DependencyProcessorImpl: DependencyProcessor {
     private let bundlePath: String?
 
     init(xcode: URL, product: URL, source: URL, intermediate: URL, bundle: URL?) {
-        xcodePath = xcode.path
-        productPath = product.path
-        sourcePath = source.path
-        intermediatePath = intermediate.path
-        bundlePath = bundle?.path
+        xcodePath = xcode.path.dirPath()
+        productPath = product.path.dirPath()
+        sourcePath = source.path.dirPath()
+        intermediatePath = intermediate.path.dirPath()
+        bundlePath = bundle?.path.dirPath()
```

Comment:
alternatively, we could create initialize URLs in `PostbuildContext` with `init(fileURLWithPath: isDirectory:)`